databricks/koalas/typedef/string_typehints.py

```diff
@@ -14,6 +14,7 @@
 # limitations under the License.
 #
 import numpy as np
+import pandas
```

Comment:
nit: seems not needed?

Reply:
Here the code path is very hacky.. so it is in order to support a case when the return type hint is `pandas.DataFrame` as an example..